docs/overview_mainnet_address.md

```diff
@@ -105,6 +105,24 @@
  - weth: [0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2](https://etherscan.io/address/0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2)
  - zrx: [0xE41d2489571d322189246DaFA5ebDe1F4699F498](https://etherscan.io/address/0xE41d2489571d322189246DaFA5ebDe1F4699F498)
 
+## Idle
```

Comment:
Unfortunately, I think your addresses in the `overview_mainnet_address.md` will get overwritten unless you write the script to insert the Idle Finance addresses. See below.

See docs build script here: https://github.com/studydefi/money-legos/blob/master/package.json#L13

Reply:
this was generated from your script 😄

Reply:
nevermind! good work, my bad.